common/misc.ml

```diff
@@ -1081,6 +1081,12 @@ let (WORD_FORALL_OFFSET_TAC:int->tactic) =
 (* Do some limited simplification in association with symbolic execution.    *)
 (* ------------------------------------------------------------------------- *)
 
+(* Ocaml reference variable for platform specific conversions *)
+let extra_word_CONV = ref [NO_CONV];;
+
+(* Delay introduction of extra conversions *)
+let apply_extra_word_convs tm = FIRST_CONV (!extra_word_CONV) tm;;
```

Comment:
This update actually looks nice - I should move SIMD-related rewriting rules to here as well if this patch is accepted.